validator.go

```diff
@@ -114,6 +114,12 @@ func (v *Validator) Require(field string) *ValidationResult {
 	}
 }
 
+func (v *Validator) RequireAll(fields ...string) {
+	for _, field := range fields {
+		v.Require(field)
+	}
```

Comment:
it seems like we would want to have a return value for this since `Require` has one... would it make sense to have this signature be:

```go
func (v *Validator) RequireAll(fields ...string) ([]*ValidationResult, bool) {
  // ...
}
```

where the boolean return value indicates validity of the set, while the slice would provide access to the results?